.circleci/config.yml

```diff
@@ -62,17 +87,7 @@ jobs:
             dbt seed --target snowflake --full-refresh
             dbt run --target snowflake --full-refresh
             dbt test --target snowflake
-      - run:
-          name: "Run Tests - BigQuery"
-          environment:
-              GCLOUD_SERVICE_KEY_PATH: "/home/circleci/gcloud-service-key.json"
-
-          command: |
-            . venv/bin/activate
-            echo `pwd`
-            cd integration_tests
-            dbt deps
-            dbt seed --target bigquery --full-refresh
-            dbt run --target bigquery --full-refresh
-            dbt test --target bigquery
+            dbt run --vars '{google_play__using_earnings: true}' --target snowflake --full-refresh
+            dbt run --vars '{google_play__using_earnings: true, google_play__using_subscriptions: true}' --target snowflake --full-refresh
```

Comment:
it looks like you're testing for 2 cases here, is there any reason why you wouldn't test for 1 (`google_play__using_earnings` and `google_play__using_subscriptions`) or 3 (one for each case)?